docs/Database_Schema.md

```diff
@@ -224,3 +224,74 @@ CREATE TABLE IF NOT EXISTS information (
     PRIMARY KEY(key)
 )
 ```
+
+----
+
+## 9. Table **transactionPool**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+| tx_hash           | BLOB      | Y  | Y        |          | The hash of transaction |
+| type              | INTEGER   |    | Y        |          | The type of transaction |
+| payload           | BLOB      |    | Y        |          | The transaction data payload |
+
+### _Create Script_
+
+```sql
+CREATE TABLE IF NOT EXISTS "transactionPool" (
+    "tx_hash"               BLOB    NOT NULL,
+    "type"                  INTEGER NOT NULL,
+    "payload"               BLOB    NOT NULL,
+    PRIMARY KEY("tx_hash")
+)
+```
+
+----
+
+## 4. Table **txInputPool**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+| tx_hash           | BLOB      | Y  | Y        |          | The hash of transaction|
+| utxo              | BLOB      | Y  | Y        |          | The hash of the UTXO to be spent|
```

Comment:
Why does this(utxo) have to be the key?

Reply:
Isn't it unique?

Reply:
How about adding in_index just in case? I think we can print it out as JSON later.

Reply:
That's a good.